# TODO make me a PoseMsg

float64[] positionRight
float64[] positionLeft
float64[] orientationRight
float64[] orientationLeft

float64[] positionAbsolute
float64[] positionRelative
float64[] orientationAbsolute
float64[] orientationRelative

float64 gripperRight
float64 gripperLeft

float64 pointTime

bool local
